json/lists.json

```diff
@@ -37,6 +37,6 @@
   "case": []
 }, {
   "countryCode": "lr",
-  "followUp": [],
+  "followUp": ["red_eyes","other_symptoms"],
```

Comment:
Yep, that is how it works right now. Not really sophisticated, but works for the moment. This is the easiest way, or else we would have to think about how we deal with exclusions and inclusions. The logic is to check for specific symptoms per country and if there are none, simply falling back to the default list.